sormas-api/src/main/java/de/symeda/sormas/api/common/DeleteDetails.java

```diff
@@ -0,0 +1,33 @@
+package de.symeda.sormas.api.common;
+
+import java.io.Serializable;
+
+public class DeleteDetails implements Serializable {
```

Comment:
```suggestion
public class DeletionDetails implements Serializable {
```

Please also rename all fields accordingly

Reply:
changed